entrypoint_debian.sh

```diff
@@ -60,6 +60,45 @@ case ${OPENHAB_VERSION} in
         cp -av "${APPDIR}/userdata.dist/." "${APPDIR}/userdata/"
       fi
 
+      # Upgrade userdata if versions do not match
+      curVersion=$(< "${APPDIR}/userdata/etc/version.properties" grep build-no | cut -d : -f 2 | tr -d '[:space]')
+      imgVersion=$(< "${APPDIR}/userdata.dist/etc/version.properties" grep build-no | cut -d : -f 2 | tr -d '[:space]')
+      
+      if [ "${curVersion}" = "${imgVersion}" ]; then
+        echo "Image build number \"${imgVersion}\" is different from userdata build number \"${curVersion}\""
+
+        # Make a backup of userdata
+        backupFile=userdata-$(date +"%FT%H:%M:%S").tar
+        tar --exclude="${APPDIR}/userdata/backup" -c -f "/tmp/${backupFile}" "${APPDIR}/userdata"
+        if [ ! -d "${APPDIR}/userdata/backup" ]; then
+          mkdir "${APPDIR}/userdata/backup"
+        fi
+        cp "/tmp/${backupFile}" "${APPDIR}/userdata/backup/"
+        echo "You can find backup of userdata in ${APPDIR}/userdata/backup/${backupFile}"
+
+        # Copy over the updated files
+        cp "${APPDIR}/userdata.dist/etc/all.policy" "${APPDIR}/userdata/etc/"
+        cp "${APPDIR}/userdata.dist/etc/branding.properties" "${APPDIR}/userdata/etc/"
+        cp "${APPDIR}/userdata.dist/etc/branding-ssh.properties" "${APPDIR}/userdata/etc/"
+        cp "${APPDIR}/userdata.dist/etc/config.properties" "${APPDIR}/userdata/etc/"
+        cp "${APPDIR}/userdata.dist/etc/custom.properties" "${APPDIR}/userdata/etc/"
+        cp "${APPDIR}/userdata.dist/etc/version.properties" "${APPDIR}/userdata/etc/"
+        cp "${APPDIR}/userdata.dist/etc/distribution.info" "${APPDIR}/userdata/etc/"
+        cp "${APPDIR}/userdata.dist/etc/jre.properties" "${APPDIR}/userdata/etc/"
+        cp "${APPDIR}/userdata.dist/etc/profile.cfg" "${APPDIR}/userdata/etc/"
+        cp "${APPDIR}/userdata.dist/etc/startup.properties" "${APPDIR}/userdata/etc"
+        cp "${APPDIR}/userdata.dist/etc/org.apache.karaf*" "${APPDIR}/userdata/etc/"
```

Comment:
Debian builds for 2.0+ are failing with
```
cp: cannot stat ‘/openhab/userdata.dist/etc/org.apache.karaf*’: No such file or directory
```
Thanks to your debug we can see it on Travis :) https://travis-ci.org/openhab/openhab-docker/jobs/325142595

Reply:
Moving the asterisk after the double quote will probably fix it.